apps/dot_ext/migrations/0014_auto_20190121_1345.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.1.2 on 2019-01-21 13:45
+
+from django.conf import settings
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dot_ext', '0013_auto_20181221_2114'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='application',
+            name='support_email',
+            field=models.EmailField(blank=True, max_length=254, null=True),
+        ),
+        migrations.AddField(
+            model_name='application',
+            name='support_phone_number',
+            field=models.CharField(blank=True, max_length=17, null=True, validators=[django.core.validators.RegexValidator(message="Phone number must be entered in the format: '+999999999'. Up to 15 digits allowed.", regex='^\\+?1?\\d{9,15}$')]),
+        ),
+        migrations.AlterField(
+            model_name='application',
+            name='id',
+            field=models.BigAutoField(primary_key=True, serialize=False),
+        ),
+        migrations.AlterField(
+            model_name='application',
+            name='redirect_uris',
+            field=models.TextField(blank=True, help_text='Allowed redirect URIs. Space or new line separated.'),
+        ),
+        migrations.AlterField(
+            model_name='application',
+            name='scope',
+            field=models.ManyToManyField(to='capabilities.ProtectedCapability'),
+        ),
+        migrations.AlterField(
+            model_name='application',
+            name='user',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='dot_ext_application', to=settings.AUTH_USER_MODEL),
+        ),
```

Comment:
Should the AlterField's be removed for the fields not being newly added?

Reply:
yes! ty